### PR TITLE
Charging stations radius

### PR DESCRIPTION
**Pull request by AlixH**
- Removed 500 kms radius for charging-stations/sites/site areas. 
- Restyle QR-Code button
- Fixed and improved Header
- Refactored Maps and changed lib

### CODE REVIEW

**Comment by AlixH** [comment]
To be beta-tested

File: src/components/map/cluster/MapCluster.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (LucasBrazi06) [inline src/components/map/cluster/MapCluster.tsx:1]: Weird the naming: ClusterMap, MapCluster LOL
Keep the same order -> ClusterMap and ClusterMapXXX ?

**Review by LucasBrazi06 — APPROVED** [review]

File: src/components/map/cluster/MapCluster.tsx (hunk 1 above)
Comment (AlixH) [reply src/components/map/cluster/MapCluster.tsx:1]: ClusterMap is the map, mapCluster is the cluster circle

Comment (LucasBrazi06) [reply src/components/map/cluster/MapCluster.tsx:1]: [low]Blanc bonnet et bonnet blanc ;-).
Find another meaningful name ;-P

**Pull request merged**